pkg/policy/api/rule_validation.go

```diff
@@ -192,17 +192,16 @@ func (pp PortProtocol) Validate() error {
 		return fmt.Errorf("Port cannot be 0")
 	}
 
-	switch strings.ToLower(pp.Protocol) {
-	case "", "any", "tcp", "udp":
-	default:
-		return fmt.Errorf("Invalid protocol %q, must be { tcp | udp }", pp.Protocol)
+	pp.Protocol, err = ParseL4Proto(string(pp.Protocol))
+	if err != nil {
+		return err
 	}
 
 	return nil
 }
 
-// Validate CIDR
-func (cidr CIDR) Validate() error {
+// Sanitize CIDR
```

Comment:
This one is actually Validating, right?

Reply:
marked private. It is validating only for now but we don't know later on. I've only kept the Rule Sanitize() public and have hidden everything so we can rename/change this as we want in the future.

Reply:
On that case can you add a pointer to the receiver?

Reply:
It's a string so it can't be a pointer.